### ISSUE TITLE
What is `$channelsTests` for in `channels.php`?

### ISSUE THREAD

**Issue by quroom** [Benjamin-Loison/YouTube-operational-API#270]
```php
['part=shorts&id=UCv_LqFI-0vMVYgNR3TeB3zQ', 'items/0/shorts', json_decode(file_get_contents('tests/part=shorts&id=UCv_LqFI-0vMVYgNR3TeB3zQ.json'), true)],
['part=community&id=UCv_LqFI-0vMVYgNR3TeB3zQ', 'items/0/community', json_decode(file_get_contents('tests/part=community&id=UCv_LqFI-0vMVYgNR3TeB3zQ.json'), true)],
['part=about&id=UCv_LqFI-0vMVYgNR3TeB3zQ', 'items/0', json_decode(file_get_contents('tests/part=about&id=UCv_LqFI-0vMVYgNR3TeB3zQ.json'), true)],
['part=snippet&id=UCv_LqFI-0vMVYgNR3TeB3zQ', 'items/0/snippet', json_decode(file_get_contents('tests/part=snippet&id=UCv_LqFI-0vMVYgNR3TeB3zQ.json'), true)],
```
As long as including this code in `$channelsTests` array, it returns response with warning in http://localhost:8080 API.
```html
<br />
<b>Warning</b>:  file_get_contents(tests/part=shorts&amp;id=UCv_LqFI-0vMVYgNR3TeB3zQ.json): Failed to open stream: No such file or directory in <b>/var/www/html/channels.php</b> on line <b>12</b><br />
<br />
<b>Warning</b>:  file_get_contents(tests/part=community&amp;id=UCv_LqFI-0vMVYgNR3TeB3zQ.json): Failed to open stream: No such file or directory in <b>/var/www/html/channels.php</b> on line <b>13</b><br />
<br />
<b>Warning</b>:  file_get_contents(tests/part=about&amp;id=UCv_LqFI-0vMVYgNR3TeB3zQ.json): Failed to open stream: No such file or directory in <b>/var/www/html/channels.php</b> on line <b>14</b><br />
<br />
<b>Warning</b>:  file_get_contents(tests/part=snippet&amp;id=UCv_LqFI-0vMVYgNR3TeB3zQ.json): Failed to open stream: No such file or directory in <b>/var/www/html/channels.php</b> on line <b>16</b><br />
{
    "kind": "youtube#channelListResponse",
    "etag": "NotImplemented",
    "items": [
     ...
    ]
}
```
When I removed it, warning is gone. and it looks like working well.
As like warning, maybe we need to put those files, but I have no idea, what files?
Could I remove that code ? I would like to know any side-effect?

I tested it in Windows WSL Ubuntu, Docker env.

**Comment by quroom** [comment]
If it should be maintained, What do you think adding at sign like `@json_decode(...)` to prevent warning?

**Comment by Benjamin-Loison** [comment]
I admit not having enough time and courage recently to finish establishing unit tests (see #24), hence they are in production in a kind of work in progress state. So in some sense the file dependent unit tests can only run by myself at this time.

It is still a bit unclear to me how to properly proceed to the best unit tests as some data value may change while we want to check their correctness, for instance `viewCount` must be a number.
For big output verifications, I use some files that I have not pushed as I am unsure of how to proceed and I like doing things cleanly.

Furthermore, I admit not having done properly the API in terms of warnings as some may be raised. As a temporary solution I encourage you to disable returning warnings in the response. If you do not know how to disable them, tell me what web-server you are using?

**Comment by Benjamin-Loison** [comment]
I solved the warning issue in #267.

**Comment by quroom** [comment]
@Benjamin-Loison Thanks for quick response :)
